src/main/java/org/broadinstitute/hellbender/tools/funcotator/DataSourceFuncotationFactory.java

```diff
@@ -142,12 +144,16 @@ public String getVersion() {
 
             // Set our overrides:
             setOverrideValuesInFuncotations(outputFuncotations);
-
-            return outputFuncotations;
         }
         else {
             return createDefaultFuncotationsOnVariant(variant, referenceContext);
         }
+
```

Comment:
The old code could have outputFuncotations of length 0 and return it, but we would have wanted a default funcotation.

This happened when all of the featureList (i.e. transcripts) were IGR, since we filter out the IGR transcripts (a modification I made later per our offline discussion)
No action.